exec/vector/src/main/java/org/apache/drill/exec/record/metadata/ColumnMetadata.java

```diff
@@ -51,19 +51,20 @@
    */
   String FORMAT_PROP = DRILL_PROP_PREFIX + "format";
 
-  /**
-   * Indicates if the column is projected. Used only for internal
-   * reader-provided schemas.
-   */
-  String PROJECTED_PROP = DRILL_PROP_PREFIX + "projected";
-
   /**
    * Indicates how to handle blanks. Must be one of the valid values defined
    * in AbstractConvertFromString. Normally set on the converter by the plugin
    * rather than by the user in the schema.
    */
   String BLANK_AS_PROP = DRILL_PROP_PREFIX + "blank-as";
 
+  /**
+   * Indicates whether to project the column in a wildcard (*) query.
+   * Special columns may be excluded from projection. Certain "special"
+   * columns may be available only when explicitly requested.
+   */
+  String EXCLUDE_FROM_WILDCARD = DRILL_PROP_PREFIX + "special";
```

Comment:
Could you please provide an example form the log reader? I did not quite understand from the description.